### ISSUE TITLE
zeligt_qi_to_df for multinomial outcomes.

### ISSUE THREAD

**Issue by christophergandrud** [IQSS/Zelig#235]
```r
libray(Zelig)
library(dplyr)

data(mexico)
sims <- zelig(vote88 ~ pristr + othcok + othsocok,
               model = "mlogit.bayes", data = mexico,
               verbose = FALSE) %>%
    setx() %>%
    sim() %>%
    zelig_qi_to_df()

head(sims)
```

Produces:

```r
##   setx_value   pristr   othcok othsocok expected_value predicted_value        NA NA.1
## 1          x 1.966887 2.195732 1.395143      0.5174956       0.2170879 0.2654165    3
## 2          x 1.966887 2.195732 1.395143      0.5758689       0.2003854 0.2237457    1
## 3          x 1.966887 2.195732 1.395143      0.5725515       0.2086853 0.2187632    1
## 4          x 1.966887 2.195732 1.395143      0.5725515       0.2086853 0.2187632    2
## 5          x 1.966887 2.195732 1.395143      0.5725515       0.2086853 0.2187632    1
## 6          x 1.966887 2.195732 1.395143      0.5483135       0.2117269 0.2399596    1
```

**Comment by christophergandrud** [comment]
Needs to also change behaviour in `qi_slimmer`.

**Comment by christophergandrud** [comment]
Resolved in: ca70b1c2e8d6762ceea5f37cb32074b2147fae79

For example:

```r
library(Zelig)
library(dplyr)

data(mexico)
sims <- zelig(vote88 ~ pristr + othcok + othsocok,
              model = "mlogit.bayes", data = mexico,
              verbose = FALSE) %>%
    setx() %>%
    sim() %>%
    zelig_qi_to_df()

head(sims)
##   setx_value   pristr   othcok othsocok expected_P(Y=1) expected_P(Y=2) expected_P(Y=3) predicted_value
## 1          x 1.966887 2.195732 1.395143       0.5174956       0.2170879       0.2654165               1
## 2          x 1.966887 2.195732 1.395143       0.5758689       0.2003854       0.2237457               1
## 3          x 1.966887 2.195732 1.395143       0.5725515       0.2086853       0.2187632               1
## 4          x 1.966887 2.195732 1.395143       0.5725515       0.2086853       0.2187632               1
## 5          x 1.966887 2.195732 1.395143       0.5725515       0.2086853       0.2187632               1
## 6          x 1.966887 2.195732 1.395143       0.5483135       0.2117269       0.2399596               1
```

and

```r
qi_slimmer(sims, qi_type = 'expected_P(Y=2)')

##   setx_value   pristr   othcok othsocok qi_ci_min qi_ci_median qi_ci_max
## 1          x 1.966887 2.195732 1.395143 0.1854314    0.2098424 0.2350889
```
